### PR TITLE
Adding support for type 20 message (Interaction message)

### PR DESCRIPTION
**Pull request by flamableassassin**
despite the docs saying otherwise type 20 is still sent on V6

### CODE REVIEW

**Comment by bsian03** [comment]
(Redo of #1143 just for clarification)

**Comment by flamableassassin** [comment]
> (Redo of #1143 just for clarification)

Oh yeah soz should have put that. ty

File: lib/Constants.js [hunk 1]
```diff
@@ -181,7 +181,8 @@ module.exports.MessageTypes = {
     GUILD_DISCOVERY_REQUALIFIED: 15,
     GUILD_DISCOVERY_GRACE_PERIOD_INITIAL_WARNING: 16,
     GUILD_DISCOVERY_GRACE_PERIOD_FINAL_WARNING: 17,
-    REPLY: 19
+    REPLY: 19,
+    APPLICATION_COMMAND: 20
```
Comment (bsian03) [inline lib/Constants.js:185]: Can you add this in the relevant TS doc as well

Comment (flamableassassin) [reply lib/Constants.js:185]: Shall do that now

File: index.d.ts [hunk 2]
```diff
@@ -1021,6 +1021,7 @@ declare namespace Eris {
       GUILD_DISCOVERY_GRACE_PERIOD_INITIAL_WARNING: 16;
       GUILD_DISCOVERY_GRACE_PERIOD_FINAL_WARNING: 17;
       REPLY: 19;
+      APPLICATION_COMMAND:20;
```
Comment (abalabahaha) [inline index.d.ts]:
```suggestion
      APPLICATION_COMMAND: 20;
```

**Review by abalabahaha — APPROVED** [review]

**Pull request merged**